src/routes/(site)/teams/team-section.svelte

```diff
@@ -36,18 +36,10 @@
       class:special-events-animation={info.id === 'special-events'}
       class:gamedev-animation={info.id === 'gamedev'}
     >
-      {#if info.id === 'general'}
-        <DiamondPicture
-          src="https://cdn.discordapp.com/icons/710225099923521558/a_f72bf9caa196d84a44ff40cdfd3f8d9a.gif?size=1024"
-          reducedMotionSrc={info.logoSrc}
-          altSrc="General Picture"
-        />
-      {:else}
-        <img
-          src={oldTerms.includes(term) ? info.oldLogoSrc ?? info.logoSrc : info.logoSrc}
-          alt={`${info.title} Team Logo`}
-        />
-      {/if}
+      <img
+        src={oldTerms.includes(term) ? info.oldLogoSrc ?? info.logoSrc : info.logoSrc}
```

Comment:
Since `DimondPicture` was removed, delete the line `import DiamondPicture from './diamond-picture.svelte';`. This will remove the website check failures!